kafka/src/kafka/impl/consumer.cpp

```diff
@@ -166,6 +166,14 @@ void Consumer::AsyncCommit() {
     }).Get();
 }
 
+OffsetRange Consumer::GetOffsetRange(const std::string& topic, std::int32_t partition) const {
+    consumer_->GetOffsetRange(topic, partition);
```

Comment:
wrap call in `utils::Async` -- we can not expose methods that blocks caller task processor OS threads